slices/libnss3.yaml

```diff
@@ -0,0 +1,18 @@
+package: libnss3
+
+essential:
+  - libnss3_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libnspr4_libs
+      - libsqlite3-0_libs
+    contents:
+      /usr/lib/*-linux-*/libnss3.so:
+      /usr/lib/*-linux-*/libnssutil3.so:
```

Comment:
Do we need the other libs as well? (It's fine to leave them behind if we don't need them for now)

```
root@855047470eb3:~# dpkg -c libnss3_2%3a3.98-1build1_amd64.deb | grep "so$"
-rw-r--r-- root/root     14560 2024-04-08 22:14 ./usr/lib/x86_64-linux-gnu/libfreebl3.so
-rw-r--r-- root/root    833688 2024-04-08 22:14 ./usr/lib/x86_64-linux-gnu/libfreeblpriv3.so
-rw-r--r-- root/root   1229736 2024-04-08 22:14 ./usr/lib/x86_64-linux-gnu/libnss3.so
-rw-r--r-- root/root    584816 2024-04-08 22:14 ./usr/lib/x86_64-linux-gnu/libnssckbi.so
-rw-r--r-- root/root    137520 2024-04-08 22:14 ./usr/lib/x86_64-linux-gnu/libnssdbm3.so
-rw-r--r-- root/root    199128 2024-04-08 22:14 ./usr/lib/x86_64-linux-gnu/libnssutil3.so
-rw-r--r-- root/root    162208 2024-04-08 22:14 ./usr/lib/x86_64-linux-gnu/libsmime3.so
-rw-r--r-- root/root    358016 2024-04-08 22:14 ./usr/lib/x86_64-linux-gnu/libsoftokn3.so
-rw-r--r-- root/root    432864 2024-04-08 22:14 ./usr/lib/x86_64-linux-gnu/libssl3.so
```

Reply:
Other libs are not needed for openjdk, but we may want to include them in a separate PR.